src/mero_conf_monitor.erl

```diff
@@ -142,16 +142,25 @@ update_clusters([ClusterDef | OldClusterDefs],
     update_clusters(OldClusterDefs, NewClusterDefs);
 update_clusters([{ClusterName, OldAttrs} | OldClusterDefs],
```

Comment:
Can we drop the assumption about the two lists here?.   So we can have clusters _added_  or _removed_  at runtime, just changing the app environment (config). 

I guess we can use set differences,   a cluster that is on the old set but not on the new:  terminate the cluster.   One  in the new but not on the old:  start it.   Things that match exactly:  do nothing   (case for changed clusters could be just let it be terminated and then started,  or had special condition to call restart_child/1 explicitly).

Handling failing cluster configs would be _not_ returning them so they are like not listed on the app environment.

Reply:
Ok, working on that.